app/src/main/cpp/skyline/services/am/controller/ICommonStateGetter.cpp

```diff
@@ -71,4 +71,9 @@ namespace skyline::service::am {
         }
         return {};
     }
+
+    // Stub implementation of SetCpuBoostMode
```

Comment:
You can remove this comment, we generally don't do this regardless of something being a stub.